lib/pages/movie_info_screen/genres_list_widget.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import '/model/genre_data_model.dart';
+import '/repository/genre_repository.dart';
+import '/constants.dart';
+
+class GenresList extends StatelessWidget {
+  static const double containerBorderRadius = 15;
+  static const double containerPadding = 5;
+  static const double containerMargin = 6;
+  static const double itemsFontSize = 15;
+  final List<num> genresIds;
+  GenreRepository genreRepository = GenreRepository();
+
+  GenresList(this.genresIds, {super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+        future: genreRepository.readJsonData(),
+        builder: (context, data) {
```

Comment:
Add the type of the parameter `(BuildContext and List<GenreMoDataModel>)` and you forgot to add a trailing comma